app/src/ui/clone-repository/cloneable-repository-filter-list.tsx

```diff
@@ -266,18 +273,13 @@ export class CloneableRepositoryFilterList extends React.PureComponent<
       )
     }
 
-    const endpointName =
-      this.props.account.endpoint === getDotComAPIEndpoint()
-        ? 'GitHub.com'
-        : getHTMLURL(this.props.account.endpoint)
-
     return (
       <div className="no-items empty-repository-list">
         <div>
-          Couldn't find any repositories for the account{' '}
+          Couldn't find any repositories for{' '}
```

Comment:
I'd like to frame this less as an error state if possible. Just a thought: 
```suggestion
          Looks like there are no repositories for{' '}
```